src/contexts/global.tsx

```diff
@@ -88,7 +88,10 @@ export const GlobalContextProvider: FC<{ overrideContextValue?: PartialDeep<Glob
     return updatedSettings
   }
 
-  const lockWallet = () => setWallet(undefined)
+  const lockWallet = () => {
+    setCurrentUsername('')
```

Comment:
After locking the wallet and opening the accounts settings, the list
would still display a "current" account. This ensures resetting
the current account when locking the wallet.